komodo/symlink/suggester/configuration.py

```diff
@@ -49,6 +49,11 @@ def update(self, release, mode):
                 else:
                     self.links[link] = repr(release)
         elif mode == "stable":
+            # Add/update deprecated link iff stable exists and has changed
+            if self.links.get(link) != release.month_alias():
```

Comment:
Why not just `self.links[link]` ?

Reply:
Because `self.links[link]` will throw an exception if no `stable` exists. The alternative is
`link in self.links[link] and self.links[link] != release.month_alias()`, but my version is terser.